_includes/help-modal.html

```diff
@@ -89,7 +89,7 @@
           <!-- Tab content -->
           <div id="MesheryUI" class="tabcontent">
             <h3>Meshery UI</h3>
-            <p>To import designs, visit <a href="https://docs.meshery.io/guides/meshery-design#import-your-designs">docs</a></p>
+            <p>To import designs, visit <a href="https://docs.meshery.io/guides/configuration-management/importing-apps#import-apps-using-meshery-ui" target="_blank">docs</a></p>
```

Comment:
@ASK-03 please change this link to: https://docs.meshery.io/extensions/importing-a-design

Reply:
Application tab is not used now

Reply:
I have done the changes.